stable/admin/templates/insights-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+
+{{- if $.Values.insights }}
+{{- if $.Values.insights.enabled }}
+{{- if $.Values.insights.plugins }}
```

Comment:
Can we combine these with `and`?

Reply:
Good catch! I've updated this in both templates.